src/aioswitcher/device/__init__.py

```diff
@@ -91,6 +96,109 @@ def value(self) -> str:
         return self._value  # type: ignore
 
 
+class ThermostatMode(Enum):
+    """Enum class representing the thermostat device's position."""
+
+    AUTO = "01", "auto"
+    DRY = "02", "dry"
+    FAN = "03", "fan"
+    COOL = "04", "cool"
+    HEAT = "05", "heat"
+
+    def __new__(cls, value: str, display: str) -> "DeviceState":
+        """Override the default enum constructor and include extra properties."""
+        new_enum = object.__new__(cls)
+        new_enum._value = value  # type: ignore
+        new_enum._display = display  # type: ignore
+        return new_enum
+
+    @property
+    def display(self) -> str:
+        """Return the display name of the mode."""
+        return self._display  # type: ignore
+
+    @property
+    def value(self) -> str:
+        """Return the value of the mode."""
+        return self._value  # type: ignore
+
+
+class ThermostatFanLevel(Enum):
+    """Enum class representing the thermostat device's fan level."""
+
+    LOW = "1", "low"
+    MEDIUM = "2", "medium"
+    HIGH = "3", "high"
+    AUTO = "0", "auto"
+
+    def __new__(cls, value: str, display: str) -> "DeviceState":
+        """Override the default enum constructor and include extra properties."""
+        new_enum = object.__new__(cls)
+        new_enum._value = value  # type: ignore
+        new_enum._display = display  # type: ignore
+        return new_enum
+
+    @property
+    def display(self) -> str:
+        """Return the display name of the fan level."""
+        return self._display  # type: ignore
+
+    @property
+    def value(self) -> str:
+        """Return the value of the fan level."""
+        return self._value  # type: ignore
+
+
+class ThermostatSwing(Enum):
+    """Enum class representing the thermostat device's swing state."""
+
+    OFF = "0", "off"
+    ON = "1", "on"  # TODO: Get actual Value
```

Comment:
```suggestion
    ON = "1", "on"
```
Verified it is working